src/app/containers/PageHandlers/withData/index.jsx

```diff
@@ -3,17 +3,17 @@ import { element, string } from 'prop-types';
 import pathOr from 'ramda/src/pathOr';
 import { dataPropType } from '../../../models/propTypes/data';
 import ErrorMain from '../../ErrorMain';
-import getPassportHome from '../../../lib/utilities/getPassportHome';
+import {
+  getPassportHome,
+  isValidPassportHome,
+} from '../../../lib/utilities/getPassportHome';
```

Comment:
```suggestion
} from '../../../lib/utilities/passport';
```
Worth a rename now it contains 2 things :)

Reply:
Done!